ui/package.json

```diff
@@ -65,5 +63,9 @@
   "readmeFilename": "README.md",
   "bugs": {
     "url": "https://github.com/Bhamni/opd/issues"
+  },
+  "dependencies": {
+    "react": "^16.12.0",
```

Comment:
@bsneha90, is it possible to use [latest react version](https://reactjs.org/versions/)

Reply:
React issues a new release every week, when this was done we did take the latest. As a product it will be difficult to keep up to speed with that.